src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,14 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def verify_name!(name)
```

Comment:
This very much sounds like an [AR validation](http://guides.rubyonrails.org/active_record_validations.html)...